pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -2523,12 +2549,9 @@ def set_initial_magnetic_moments(self, magmoms=None):
                 self.add_tag(spin=None)
             for ind, spin in enumerate(magmoms):
                 self.spin[ind] = spin
-        # ASE part
-        if magmoms is None:
-            self.set_array('initial_magmoms', None)
-        else:
-            magmoms = np.asarray(magmoms)
-            self.arrays['initial_magmoms'] = magmoms
+        if magmoms is not None:
```

Comment:
Isn't this line redundant? This means, if `magmoms` is not `None`, it's explicitly set to `None` and if `magmoms` is `None`, it's set to `magmoms`, which is `None`.

Reply:
This should now be fixed